UnitsNet.Tests/UnitSystemTests.cs

```diff
@@ -149,5 +150,55 @@ public void SIUnitSystemHasCorrectBaseUnits()
             Assert.Equal(AmountOfSubstanceUnit.Mole, UnitSystem.SI.BaseUnits.Amount);
             Assert.Equal(LuminousIntensityUnit.Candela, UnitSystem.SI.BaseUnits.LuminousIntensity);
         }
+
+        [Fact]
+        public void GetDefaultUnitInfoThrowsExceptionForUndefinedQuantity()
+        {
+            Assert.Throws<ArgumentException>(() => UnitSystem.SI.GetDefaultUnitInfo(QuantityType.Undefined));
+        }
+
+        [Fact]
+        public void GetDefaultUnitInfoReturnsNullForQuantitiesWithNoDefaultUnits()
+        {
+            // TODO do we expect to preserve this behavior?
+            // AmplitudeRatio might be unitless- but there are (more than one) ways to express ratios. 
+            Assert.Null(UnitSystem.SI.GetDefaultUnitInfo(AmplitudeRatio.QuantityType));
+        }
+
+        [Fact]
+        public void WithDefaultUnitThrowsIfQuantityTypeIsUndefined()
+        {
+            Assert.Throws<ArgumentException>(() => UnitSystem.SI.WithDefaultUnit(QuantityType.Undefined, null));
+        }
+
+        [Fact]
+        public void WithDefaultUnitUsesOldBaseUnitsIfNotSpecified()
```

Comment:
Please use naming convention of `WhatIsTested_WhatAreTheInputsOrConditions_ExpectedOutcome`.
In this case, `WithDefaultUnit_GivenNullForBaseUnits_ReturnsUnitSystemWithOldBaseUnits` or something like that. 

I don't fully understand the intent of this test case, however. Aren't we removing BaseUnits concept altogether?